tests/scripts/run-throughput-perf.py

```diff
@@ -417,6 +430,9 @@ def main(args):
                 "--config",
                 "OptLevel",
                 opt_level,
+                "--config",
+                "JitName",
+                jit_name,
```

Comment:
I've talked to @RussKeldorph about this, and I'm actually pretty ok with the JitName config option.